dotCMS/src/integration-test/java/com/dotcms/auth/providers/jwt/factories/ApiTokenAPITest.java

```diff
@@ -357,7 +361,26 @@ public void test_expired_ApiToken() throws Exception {
     @Test
     public void test_user_must_be_active_to_validate_ApiToken() throws Exception {
 
-        User user = new UserDataGen().nextPersisted();
+        final Company company = new CompanyDataGen()
```

Comment:
fwiw, we don't really support multiple companies.  Not sure what the ramifications of this are.